vignettes/adrs.Rmd

```diff
@@ -393,6 +393,28 @@ dataset_vignette(
 )
 ```
 
+Note that the above gives pre-defined `AVAL` values of:
+`"CR" ~ 1`, `"PR" ~ 2`, `"SD" ~ 3`, `"NON-CR/NON-PD" ~ 4`, `"PD" ~ 5`, `"NE" ~ 6`, `"MISSING" ~ 7`.
+
+If you'd like to provide your own company-specific ordering here you could do this as follows:
+
+```{r, eval=FALSE}
+aval_resp_new <- function(arg) {
+  case_when(
+    arg == "CR" ~ 7,
+    arg == "PR" ~ 6,
+    arg == "SD" ~ 5,
+    arg == "NON-CR/NON-PD" ~ 4,
+    arg == "PD" ~ 3,
+    arg == "NE" ~ 2,
+    arg == "MISSING" ~ 1,
+    TRUE ~ NA_real_
+  )
+}
+```
+
+Then add the additional argument `aval_fun = aval_resp_new` to the above `derive_param_bor()` call.
+
```

Comment:
@rossfarrugia may be I miss-understood what Stefan mentioned in the call, the internal processing of the function still remain on the pre-defined `AVAL` values of:
`"CR" ~ 1`, `"PR" ~ 2`, `"SD" ~ 3`, `"NON-CR/NON-PD" ~ 4`, `"PD" ~ 5`, `"NE" ~ 6`, `"MISSING" ~ 7`.
But the AVAL values can be changed in the final output data using aval_resp_new function. I think that's fine but if we can specify it ( if I am right).  also on the statement above  "Then add the additional argument `aval_fun = aval_resp_new` to the above `**derive_param_bor(**)` call.", yo mean "derive_param_confirmed_bor().

Reply:
Will add extra sentence to explain: Be aware that this will only impact the `AVAL` mapping, not the derivation of BOR in any way - as the function derivation relies only on `AVALC` here.

`derive_param_bor` is correct as this section was added below that function. the confirmed section comes later in the vignette.